MediaRecorder.bs

```diff
@@ -580,6 +580,8 @@ dictionary MediaRecorderOptions {
   unsigned long audioBitsPerSecond;
   unsigned long videoBitsPerSecond;
   unsigned long bitsPerSecond;
+  BitrateMode audioBitrateMode;
+  BitrateMode videoBitrateMode;
```

Comment:
What about default values?

Reply:
I've updated the PR with default values now.